### PR TITLE
update version of dependabot

### PR DESCRIPTION
**Pull request by alexjoybc**
# Description

This PR includes the following proposed change(s):

- Update the version of dependabot

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] 0Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
-  - package_manager: "java:maven"
+updates:
+  # Mvn dependencies
+  - package-ecosystem: "maven"
     directory: "/src/backend"
-    update_schedule: "daily"
-  - package_manager: "javascript"
+    schedule:
+      interval: "daily"
+      time: "18:00"
```
Comment (akroon3r) [inline .github/dependabot.yml:8]: 6am not 6pm :P

Comment (alexjoybc) [reply .github/dependabot.yml:8]: UTC ;-)

Comment (alexjoybc) [reply .github/dependabot.yml:8]: it should run at 11 to test it today ;-) if you merge before ahahahhahahaahw

Comment (akroon3r) [reply .github/dependabot.yml:8]: SNEAKY

**Pull request merged**